deployment/helm/synopsys-alert/README.md

```diff
@@ -500,6 +501,42 @@ For the on-premise database deployment a second Persistent Volume must be create
 - Configure Postgres admin username and password following the instructions outlined here: [Using On-Premise Database Configuration](#using-on-premise-database-configuration)
 
 ### External Postgres Database
+
+#### External Postgres Database Requirements
+- Postgres Version: 12
+- Extension: uuid-ossp (Note: this should be installed prior to creating the database)
+- Schemas: public, alert
+- Roles/Privileges: Alert requires two sets of Postgres Privileges. One set of privileges is necessary for initializing and upgrading the database. The other set (which is a subset of the first) is for reading and writing data when the
+  application is running. Although it is possible to use one role to handle both of these cases, it is recommended to separate them.
+    - Ensure the DB roles have the public schema on their search_path(s):
+      ```bash
+      ALTER ROLE <user> SET search_path = "$user", public;
+      ```
+    - The Alert Admin role should have the following privileges on all objects in the alert schema (or the schema/database when relevant):
+        - SELECT
+        - INSERT
+        - CREATE
+        - UPDATE
+        - DELETE
+        - TRUNCATE
+        - REFERENCES
+        - TRIGGER
+        - TEMPORARY
+        - EXECUTE
+        - USAGE
+    - The Alert Admin role should have the following privileges on all objects in the public schema (or the schema/database when relevant):
+        - SELECT
+        - INSERT
+        - UPDATE
+        - DELETE
+        - EXECUTE
+    - The Alert User role should have the following privileges on all objects in the alert schema (or the schema/database when relevant):
+        - SELECT
+        - INSERT
+        - UPDATE
+        - DELETE
+        - EXECUTE
+
 - On the external database create a user to own the Alert database
```

Comment:
Are the steps from line 540 - 554 still correct? They seem  to conflict with our documented requirements.

Reply:
@psantos1113 Is DB ownership required? It doesn't seem to be based on our liquibase. We just need to be able to perform the necessary operations in our liquibase scripts and at runtime.

Reply:
They don't have to create a separate user or database for Alert.  It isn't required based on what you've added to the document.  If they have an existing database and user with the permissions, then that can be used.  The steps to create a user and a database were here to help simplify the setup process for a user of Alert and separate the Alert data from any other database data.  Ultimately it's the database administrators choice.